src/main/java/com/google/sps/data/GivrUser.java

```diff
@@ -104,6 +115,38 @@ public static void updateUserInDatastore(String identifyingProperty, String iden
     datastore.put(entity);
   }
 
+  public void updateModeratingOrgs() {
```

Comment:
Add a comment that this updates the organizations (not reading the moderating organizations for the user which is how I approached it as first.) In general, all public functions should have some documentation.